pkg/controllers/nsxserviceaccount/nsxserviceaccount_controller.go

```diff
@@ -104,7 +176,13 @@ func (r *NSXServiceAccountReconciler) Reconcile(ctx context.Context, req ctrl.Re
 					return ResultRequeue, err
 				}
 			}
-			r.StatusUpdater.IncreaseDeleteSuccessTotal()
```

Comment:
Curious why originally this is `IncreaseDeleteSuccessTotal()`. Not sure @gran-vmv knows this.

Reply:
This is already discussed in https://github.com/vmware-tanzu/nsx-operator/pull/1019#discussion_r1951922348